plugin/trino-openlineage/src/test/java/io/trino/plugin/openlineage/TestOpenLineageEventListenerMarquezIntegration.java

```diff
@@ -109,17 +109,17 @@ void testCreateTableAsSelectFromView()
             URI trino = new URI(trinoURI);
 
             String expectedNamespace = URLEncoder.encode(format("trino://%s:%s", trino.getHost(), trino.getPort()), UTF_8);
-            String expectedQueryId = URLEncoder.encode(queryId, UTF_8);
+            String expectedJobName = URLEncoder.encode(queryId, UTF_8);
 
-            checkJobRegistration(client, expectedNamespace, expectedQueryId);
+            checkJobRegistration(client, expectedNamespace, expectedJobName);
         });
     }
 
-    private void checkJobRegistration(HttpClient client, String expectedNamespace, String expectedQueryId)
+    private void checkJobRegistration(HttpClient client, String expectedNamespace, String expectedJobName)
             throws URISyntaxException, IOException, InterruptedException
     {
         HttpRequest requestJob = HttpRequest.newBuilder()
-                .uri(new URI(marquezURI + "/api/v1/namespaces/" + expectedNamespace + "/jobs/" + expectedQueryId))
+                .uri(new URI(marquezURI + "/api/v1/namespaces/" + expectedNamespace + "/jobs/" + expectedJobName))
```

Comment:
I think `expectedQueryId` is more suitable here, request should bind to the queryId not the name